### PR TITLE
 	Bug 1208086 - Hide links to whatsdeployed

### PR DESCRIPTION
**Pull request by wlach**
It currently doesn't do anything useful. Commenting it out instead of eliminating
it just in case someone wants to bring it back.

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1004)

### CODE REVIEW

File: ui/userguide.html [hunk 1]
```diff
@@ -641,12 +641,14 @@
...
       </div>
+      <!-- Disabling whatsdeployed for now -- https://bugzilla.mozilla.org/show_bug.cgi?id=1208086 -->
+      <!--
       <div class="col-xs-6">
         <a class="midgray pull-right"
            href="https://whatsdeployed.paas.allizom.org/?owner=mozilla&amp;repo=treeherder&amp;name[]=Heroku-prototype&amp;url[]=https://treeherder-heroku.herokuapp.com/revision.txt&amp;name[]=Stage&amp;url[]=https://treeherder.allizom.org/revision.txt&amp;name[]=Prod&amp;url[]=https://treeherder.mozilla.org/revision.txt">What's Deployed?</a>
       </div>
     </div>
-
+    -->
```
Comment (tojonmz) [inline ui/userguide.html]: I haven't tested locally, but should this `-->` be up one line?

Comment (wlach) [inline ui/userguide.html]: You mean on the other side of the closing div? I don't think it really matters that much tbh.

Comment (edmorley) [inline ui/userguide.html]: The markup is not valid as is, no?

Comment (wlach) [inline ui/userguide.html]: I don't understand, what's the issue with it? It works fine in my testing.

Comment (edmorley) [inline ui/userguide.html]: The browser is likely compensating for the mismatched `<div>`s.

With this PR there are more `<div>`s than `</div>`s.

Comment (wlach) [inline ui/userguide.html]: Ah gotcha, now I see! Sorry for the confusion, I'll fix before pushing.

**Pull request merged**